.github/workflows/release.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
     strategy:
       matrix:
         python-version: [3.8, 3.9, "3.10", 3.11, 3.12, 3.13]
-        os: [ubuntu-latest, macos-13, macos-14, windows-2022]  # macos-13: x86, macos-14: arm64
+        os: [ubuntu-latest, ubuntu-24.04-arm, macos-13, macos-14, windows-2022]  # macos-13: x86, macos-14: arm64
```

Comment:
Could we also add `ubuntu-24.04-arm` to `ci.yaml`?

Reply:
Sure! That means I'll have to remove the x86 hard-coding in `_run_cpp`